examples/base/main.tf

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
This example doesn't really help that much, but if we want to keep it I'd suggest calling it "setup" instead of base.

Reply:
Ok, will do.